ironfish/src/rpc/routes/wallet/signTransaction.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { UnsignedTransaction } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { AssertHasRpcContext } from '../rpcContext'
+import { getAccount } from './utils'
+
+export type SignTransactionRequest = {
+  unsignedTransaction: string
+  account?: string
```

Comment:
Should `wallet/signTransaction` take an account?

We've deprecated the account field from `postTransaction` because you can determine the sender account from the raw transaction. Is it possible to determine the sender from an unsigned transaction?

Reply:
Done!